tests/ctst/steps/replication.ts

```diff
@@ -83,12 +88,12 @@ When('I run the job to replicate existing objects with status {string}',
         await createAndRunPod(this, podManifest);
     });
 
-Then('the object should eventually be replicated',
-    async function (this: Zenko) {
+Then('the object should eventually {string} replicated', { timeout: 360_000 },
```

Comment:
nit: alternatively, we could make this a bit more generic by specifying the replication status in the test?

```suggestion
Then('the object replication state should eventually be {string}', { timeout: 360_000 },
```
where string can be COMPLETED or FAILED (or PENDING, but this could create flakoness)

It avoids the "mapping", not sure if it is more useful or it makes the tests more easier to read...

Reply:
I will reconsider this one with the next pr which adds another test, because this new test will introduce a third new status variable